### ISSUE TITLE
Duplicate phpName table attribute should throw error

### ISSUE THREAD

**Issue by maroonraccoon** [propelorm/Propel2#1008]
Generated classes are overwritten during `model:build` when there are tables with duplicate _phpName_ attributes leading to unexpected behaviour during development.

An error should be thrown instead of overwriting classes.

[Sandbox](http://sandbox.propelorm.org/5cd561e)

**Comment by marcj** [comment]
Yes, I agree, a simple exception showing that you're currently having two tables with the same class name in the schema should be valuable.
